control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1323,6 +1358,103 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 	return nil
 }
 
+func (r *HostedControlPlaneReconciler) reconcileKubevirtCSIDriver(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
+	// var crdResource = schema.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1", Resource: "customresourcedefinitions"}
+	// r.TenantDynamicClient.Resource(crdResource).Create(ctx, kubevirt_assets.CsiDriverCRD.(*unstructured.Unstructured), metav1.CreateOptions{})
+
+	// apixClient, err := apixv1beta1client.NewForConfig(r.TenantRestConfig)
+	// if err != nil {
+	// 	return fmt.Errorf("failed to build rest config: %w", err)
+	// }
+	// crds := apixClient.CustomResourceDefinitions()
+	// crds.Cr
+	infraNamespace := hcp.Namespace
+	infraUrl := r.RestConfig.APIPath
+	infraCertificateAuthorityData := string(r.RestConfig.CAData)
+	infraToken := "TODO"
+	infraKubeconfigSecretName := "tenant-controller-kubeconfig"
```

Comment:
var name says infra, the value says tenant

Reply:
Yes, because the object is stored in infra and contains credentials for the tenant.